cpp/arcticdb/storage/mongo/mongo_mock_client.cpp

```diff
@@ -0,0 +1,201 @@
+/* Copyright 2023 Man Group Operations Limited
```

Comment:
```suggestion
/* Copyright 2024 Man Group Operations Limited
```

Reply:
Fixed in [ea60ab1](https://github.com/man-group/ArcticDB/pull/1395/commits/ea60ab11b0de9987e1146fa249374be97efd0934)